do-catalog/src/styles/components/_checkbox.scss

```diff
@@ -0,0 +1,117 @@
+.checkbox {
+  position: relative;
+
+  .checkbox-decoration {
+    display: flex;
+    position: relative;
+    width: 24px;
+    height: 24px;
+    overflow: hidden;
+    border: 1px solid $checkbox__border-color;
+    border-radius: 2px;
+    pointer-events: none;
+  }
+
+  .checkbox-check {
+    transform-origin: 50% 50%;
+    stroke-width: 2;
+    stroke-dasharray: 48;
+    stroke-dashoffset: 48;
+    stroke: $checkbox-stroke__color;
+  }
+
+  .checkbox-decorationMedia {
+    display: flex;
+    position: relative;
+    width: 18px;
+    margin-right: auto;
+    margin-left: auto;
+    transform: translateY(3px);
+  }
+
+  .checkbox-input {
+    position: absolute;
+    -webkit-appearance: none;
+    appearance: none;
+    top: 0;
+    left: 0;
+    width: 24px;
+    height: 24px;
+    margin: 0;
+    padding: 0;
+    border-radius: 3px;
+    background: $checkbox__bg-color;
+    cursor: pointer;
+
+    &::before {
+      content: ' ';
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      // background-image: url('../assets/icons/common/check-hover.svg');
+      background-repeat: no-repeat;
+      background-position: center;
+    }
+
+    &::after {
+      content: '';
+      position: absolute;
+      top: -3px;
+      left: -3px;
+      width: calc(100% + 6px);
+      height: calc(100% + 6px);
+    }
+
+    &:hover + .checkbox-decoration {
+      border: 1px solid $checkbox__border-color--hover;
+    }
+
+    &.is-selected,
+    &:checked {
+      + .checkbox-decoration {
+        border: 1px solid $checkbox__border-color--hover;
+
+        &::before {
+          content: "";
```

Comment:
Double quote to single

Reply:
This is part of the code I imported from the Dashboard, but I have updated, thanks!